pkg/template/cloudconfig.go

```diff
@@ -28,29 +25,6 @@ type CloudConfigConfig struct {
 	Template string
 }
 
-func DefaultCloudConfigConfig() CloudConfigConfig {
-	return CloudConfigConfig{
-		Params:   Params{},
-		Template: "",
-	}
-}
-
-func DefaultParams() Params {
```

Comment:
This feels like a problematic function for me. It allows operators to use the library without considering all of the parameters that are passed in. For this reason, I decided to remove it while working on this PR. Interested to hear what others think.

Reply:
We got rid of these functions in other places - so totally fine with me.

Reply:
Sounds good. I will also add this decision to `fmt` so we can stay aligned across projects.